TSOClient/tso.client/UI/Profile/UIJobInfo.cs

```diff
@@ -0,0 +1,377 @@
+﻿using FSO.Client.UI.Controls;
+using FSO.Client.UI.Framework;
+using FSO.Common.Utils;
+using FSO.Files;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FSO.Client.UI.Profile
```

Comment:
Did you mean "model"? This should go in FSO.Client.UI.Panels, or some subfolder in there.

Reply:
I updated the using declarations to a more appropriate; refined path.